imap_processing/hit/l1a/hit_l1a.py

```diff
@@ -0,0 +1,208 @@
+"""Decommutate HIT CCSDS data and create L1a data products."""
+
+import logging
+from collections import defaultdict
+from dataclasses import fields
+from enum import IntEnum
+
+import numpy as np
+import xarray as xr
+
+from imap_processing import decom, imap_module_directory, utils
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.hit import hit_cdf_attrs
+from imap_processing.hit.l0.data_classes.housekeeping import Housekeeping
+
+logger = logging.getLogger(__name__)
+
+
+class HitAPID(IntEnum):
+    """
+    HIT APID Mappings.
+
+    Attributes
+    ----------
+    HIT_AUT : int
+        Autonomy
+    HIT_HSKP: int
+        Housekeeping
+    HIT_SCIENCE : int
+        Science
+    HIT_IALRT : int
+        I-ALiRT
+    HIT_MEMDUMP : int
+        Memory dump
+    """
+
+    HIT_AUT = 1250  # Autonomy
+    HIT_HSKP = 1251  # Housekeeping
+    HIT_SCIENCE = 1252  # Science
+    HIT_IALRT = 1253  # I-ALiRT
+    HIT_MEMDUMP = 1255  # Memory dump
+
+
+def hit_l1a_data(packet_file: str):
+    """
+    Process HIT L0 data into L1A data products.
+
+    Parameters
+    ----------
+    packet_file : str
+        Path to the CCSDS data packet file.
+
+    Returns
+    -------
+    dict
+        List of file paths to CDF data product files
+    """
+    # Decom, sort, and group packets by apid
+    packets = decom_packets(packet_file)
+    sorted_packets = sorted(packets, key=lambda x: x.data["SHCOARSE"].derived_value)
```

Comment:
Use the global utility function here too?
https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/1efb973eb1830c4e3407c84e1199d8b83c1a7899/imap_processing/utils.py#L19
```suggestion
    sorted_packets = sort_by_time(packets, "SHCOARSE")
```

Reply:
Nice! I missed this global function. Thanks for the suggestion